dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-record-correction-details.test.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  FeeRecordEntityMockBuilder,
+  FeeRecordCorrectionEntityMockBuilder,
+  mapReasonsToDisplayValues,
+  UtilisationReportEntityMockBuilder,
+  PENDING_RECONCILIATION,
+  FEE_RECORD_STATUS,
+  RECORD_CORRECTION_REASON,
+} from '@ukef/dtfs2-common';
+import { format } from 'date-fns';
+import { getRecordCorrectionDetails } from './get-record-correction-details';
+
+describe('get-record-correction-details', () => {
+  const feeRecordId = 11;
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('when no record corrections are provided', () => {
+    const uploadedUtilisationReport = UtilisationReportEntityMockBuilder.forStatus(PENDING_RECONCILIATION).build();
+    const feeRecord = FeeRecordEntityMockBuilder.forReport(uploadedUtilisationReport).withId(feeRecordId).withCorrections([]).build();
+    uploadedUtilisationReport.feeRecords = [feeRecord];
```

Comment:
Since the report does not matter to the function under test we can simplify the set up to:
```suggestion
    const feeRecord = new FeeRecordEntityMockBuilder().withId(feeRecordId).withCorrections([]).build();
```
Similarly in the below describe blocks